cpp/demo/volume/main.cpp

```diff
@@ -65,13 +66,20 @@ int main(int argc, char* argv[])
   MatAssemblyEnd(B.mat(), MAT_FINAL_ASSEMBLY);
   t1.stop();
 
-  double normA;
-  MatNorm(A.mat(), NORM_FROBENIUS, &normA);
+  MatInfo info;
+  MatGetInfo(A.mat(), MAT_LOCAL, &info);
 
-  double normB;
-  MatNorm(B.mat(), NORM_FROBENIUS, &normB);
+  double* A_array;
+  MatSeqAIJGetArray(A.mat(), &A_array);
+  auto _A = xt::adapt(A_array, info.nz_allocated, xt::no_ownership(),
+                      std::vector<std::size_t>{std::size_t(info.nz_allocated)});
 
-  assert(xt::isclose(normA, normB));
+  double* B_array;
```

Comment:
Should we make a utils function for this? I feel like this is something that we will repeat in alot of verification code.

Reply:
Done!